skin/base.css

```diff
@@ -717,26 +721,30 @@
 }
 
 #TabsToolbar > #new-tab-button {
-  display: inline;
-}
-
-#TabsToolbar > #new-tab-button {
+  display: -moz-box !important;
   margin: 0 4px !important;
+  padding: 0 6px !important;
+  min-width: 40px !important;
+  -moz-box-align: center !important;
   -moz-box-orient: horizontal !important;
+  -moz-box-pack: end !important;
+  flex: 0 20 auto !important;
 }
 
 #TabsToolbar #new-tab-button .toolbarbutton-icon {
-  margin-right: 7px !important;
+  margin: 0 !important;
+  margin-right: 6px !important;
+  padding: 0 !important;
 }
 
 #TabsToolbar #new-tab-button .toolbarbutton-text {
-  display: -moz-box;
+  display: inline !important;
   padding: 0 !important;
-  margin: 0 !important;
+  /* margin: 0 !important; */
```

Comment:
Let's just remove this instead of commenting it out.  We have git history if we want to get it back.  🙂